pkg/machines/libvirt-dbus.js

```diff
@@ -506,6 +506,13 @@ LIBVIRT_DBUS_PROVIDER = {
             };
         }
 
+        /* Initial all resources to empty objects before loading them */
+        unknownConnectionName(initInterfaces);
```

Comment:
How does that actually work? `unknownConnectionName(...)` just returns a function (dispatch => blah), and as you ignore the return value, this looks like a complete no-op.